github-event-types.md : add missed event object common properties into list (#18167)
github/docs: content/developers/webhooks-and-events/events/github-event-types.md

| `repo.name` | The name of the repository, which includes the owner and repository name. For example, `octocat/hello-world` is the name of the `hello-world` repository owned by the `octocat` personal account. |
| `repo.url` | The REST API URL used to retrieve the repository object, which includes additional repository information. |
| `payload` | The event payload object is unique to the event type. See the event type below for the event API `payload` object. |

> | `public` | The property shows whether the event can be shown to all API callers or only authorized ones. |
> | `created_at` | The date and time when the event was triggered. It is formatted according to ISO 8601. |
> | `org` | The organization that was chosen by the actor to perform action that triggers the event.<br />_The property appears in the event object only if it is applicable._ |
> | `org.id` | The unique identifier for the organization. |
> | `org.login` | The name of the organization. |
> | `org.gravatar_id` | The unique identifier of the Gravatar profile for the organization. |
> | `org.url` | The REST API URL used to retrieve the organization object, which includes additional organization information. |
> | `org.avatar_url` | The URL of the organization's profile image. |

Reviewer: ```suggestion
| `public` | Whether the event is visible to all users. |
| `created_at` | The date and time when the event was triggered. It is formatted according to ISO 8601. |
| `org` | The organization that was chosen by the actor to perform action that triggers the event.<br />_The property appears in the event object only if it is applicable._ |
| `org.id` | The unique identifier for the organization. |
| `org.login` | The name of the organization. |
| `org.gravatar_id` | The unique identifier of the Gravatar profile for the organization. |
| `org.url` | The REST API URL used to retrieve the organization object, which includes additional organization information. |
| `org.avatar_url` | The URL of the organization's profile image. |
```
